flow/flow.go

```diff
@@ -828,6 +841,17 @@ func GetPortMACAddress(port uint16) [common.EtherAddrLen]uint8 {
 	return low.GetPortMACAddress(port)
 }
 
+// SetIPForPort sets IP for specified port if it was created. Not thread safe
+func SetIPForPort(port uint16, ip uint32) bool {
```

Comment:
Comment should explain the return value because it is not clear what bool means here. An error in case of failure would be more appropriate.